Descent3/init.cpp

```diff
@@ -1343,26 +1241,26 @@ void LoadGameSettings() {
     strcpy(Game_gauge_usefile, GameArgs[tt_arg + 1]);
   }
 
-  Detail_settings.Specular_lighting = false;
+  Detail_settings.Specular_lighting = true;
```

Comment:
This is not cleanup. This is changing defaults. Same for bitdepth and render distance. It needs a separate PR motivating the changes.

Reply:
For bitdepth, 16bit is no longer supported by anything isn't it? I can make it separate but bumping the defaults to new minimums seems reasonable.

Reply:
I have not studied the implications of the bit depth change in this file. It will require more testing and understanding of the code, and is outside of the scope of a code cleanup MR anyway.

Reply:
Fair enough.